pyomo/util/subsystems.py

```diff
@@ -78,6 +83,21 @@ class TemporarySubsystemManager(object):
     """
 
     def __init__(self, to_fix=None, to_deactivate=None, to_reset=None):
+        """
+        Arguments
+        ---------
+        to_fix: List of var data objects that should be temporarily fixed.
+                These are restored to their original status on exit from
+                this object's context manager.
+        to_deactivate: List of constraint data objects that should be
+                       temporarily deactivated. These are restored to their
+                       original status on exit from this object's context
+                       manager.
+        to_reset: List of var data objects that should be reset to their
+                  original values on exit from this object's context
+                  context manager.
```

Comment:
Can you reformat this to include the type?

Reply:
Updated